docs/source/run_model_in_simulation.rst

```diff
@@ -0,0 +1,204 @@
+    max_pool (MaxPooling)                    (1, 14, 14, 32)                  0
+    ____________________________________________________________________________________________
+    reorder (Reorder)                        (1, 6272)                        0
+    ____________________________________________________________________________________________
+    mat_mul (InnerProduct)                   (1, 10)                        62720
+    ____________________________________________________________________________________________
+
+This means the model has been successfully loaded in the simulation. Then we
+will see the following right after the network summary::
+
+    ======================================================
+    Tiling operators of the network...
+    ======================================================
+    Tiling conv (Convolution3d).
+    Tiling data (Data).
+    Tiling data_1 (Data).
+    Tiling data_2 (Data).
+    Tiling mat_mul (InnerProduct).
+    Tiling max_pool (MaxPooling).
+    Tiling reorder (Reorder).
+
+This shows that SMAUG is performing pre-tiling procedures for each operator -
+read-only tensors such as weights can be tiled before the actual layer-by-layer
+network execution. After this, we can see the simulation switches to use the OoO
+CPU model::
+
+    Switched CPUS @ tick 29955086000
+    switching cpus
+
+This means the initialization is done and SMAUG will start scheduling operators
+of the model. We can see::
+
+    ======================================================
+    Scheduling operators of the network...
+    ======================================================
+    Scheduling data (Data).
+    Scheduling data_1 (Data).
+    Scheduling data_2 (Data).
+    Scheduling conv (Convolution3d).
+
+As the :code:`conv` operator needs to invoke the convolution engine, we begin
+to see the Aladdin simulation logs. After each invocation of the hardware
+block, results are printed::
+
+    ===============================
+           Aladdin Results
+    ===============================
+    Running : ./outputs/nnet_fwd
+    Top level function: smv_conv3d_nhwc_vec_fxp
+    Cycle : 89771 cycles
+    Upsampled Cycle : 0 cycles
+    Avg Power: 132.997 mW
+    Idle FU Cycles: 24432 cycles
+    Avg FU Power: 111.49 mW
+    Avg FU Dynamic Power: 102.124 mW
+    Avg FU leakage Power: 9.36592 mW
+    Avg MEM Power: 21.5071 mW
+    Avg MEM Dynamic Power: 2.5961 mW
+    Avg MEM Leakage Power: 18.911 mW
+    Total Area: 1.91503e+06 uM^2
+    FU Area: 775140 uM^2
+    MEM Area: 1.13989e+06 uM^2
+    Num of Multipliers (32-bit): 37
+    Num of Adders (32-bit): 138
+    Num of Bit-wise Operators (32-bit): 12
+    Num of Shifters (32-bit): 21
+    Num of Registers (32-bit): 1798
+    ===============================
+          Aladdin Results
+    ===============================
+
+Depending on the machine on which the simulation runs, it takes about 10 mins
```

Comment:
Rephrase: "How long the simulation will take depends on the machine. On my Intel Core i7 9850H, it takes about 10 minutes".

Perhaps it might be better to delay discussion of sampling until here, but it should be here.

Reply:
Done.